configure.py

```diff
@@ -53,7 +53,8 @@
 valid_os = ('win', 'mac', 'solaris', 'freebsd', 'openbsd', 'linux',
             'android', 'aix', 'cloudabi')
 valid_arch = ('arm', 'arm64', 'ia32', 'mips', 'mipsel', 'mips64el', 'ppc',
-              'ppc64', 'x32','x64', 'x86', 'x86_64', 's390x', 'riscv64')
+              'ppc64', 'x32','x64', 'x86', 'x86_64', 's390x', 'riscv64',
```

Comment:
once upon a time these entries were sorted, it's not critical but maybe while you're in here you could fix up these last 3 entries